cmd/headerSync.go

```diff
@@ -61,7 +62,8 @@ func init() {
 }
 
 func backFillAllHeaders(blockchain core.BlockChain, headerRepository datastore.HeaderRepository, missingBlocksPopulated chan int, startingBlockNumber int64) {
```

Comment:
Hm, I think this function will be called repeatedly over the lifetime of the container. Do you think there's any way we could set it up so that we write the health check only once?

Reply:
Good catch! I wanted to make sure that we can connect to the blockchain before writing this file, but it looks like we're also called `blockchain.LastBlock` when we're validating the command args - so I moved `statusWriter.Write()` to after that succeeds, so it should just be called once at startup.